autosklearn/automl.py

```diff
@@ -128,9 +127,46 @@ def __init__(self,
         self._resampling_strategy = resampling_strategy
         self._resampling_strategy_arguments = resampling_strategy_arguments \
             if resampling_strategy_arguments is not None else {}
+        if self._resampling_strategy not in ['holdout',
```

Comment:
These were so far part of the fit method, is there a reason you moved them to init?

Reply:
If they are provided in the estimator definition, shouldn't the check be done there?
Because if someone defines an incorrect resampling strategy when doing the estimator, he will find out during fit, but then, he will have to redefine the estimator?

Reply:
Hm, true, but then there are a lot of other checks within `fit` that you did not move. Could you please move them for consistency?